### PR TITLE
Ci updates

### PR DESCRIPTION
**Pull request by matzew**
* adding source image (like we have on `knative-eventing`)
* just use 4.5 and 4.6 on the CI runs

### CODE REVIEW

File: openshift/ci-operator/source-image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1 @@
+FROM src
```
Comment (devguyio) [inline openshift/ci-operator/source-image/Dockerfile:1]: New line?

**Review by devguyio — APPROVED** [review]
just needs a new line in the Dockerfile

File: openshift/ci-operator/source-image/Dockerfile (hunk 1 above)
Comment (matzew) [reply openshift/ci-operator/source-image/Dockerfile:1]: I copied the file from eventing :man_shrugging:

**Comment by matzew** [comment]
/hold

@mgencur please unhold after your review

File: openshift/ci-operator/source-image/Dockerfile (hunk 1 above)
Comment (matzew) [inline openshift/ci-operator/source-image/Dockerfile:1]: @mgencur do we really need this?

We already have some hooks in there to generate the SRC-image:

https://github.com/matzew/eventing-sources/blob/ci_updates/openshift/ci-operator/generate-ci-config.sh#L13-L14

Comment (mgencur) [reply openshift/ci-operator/source-image/Dockerfile:1]: No. We don't need this. The lines you mentioned should take the src image directly

**Pull request merged**